### PR TITLE
Implement custom group model

### PR DESCRIPTION
**Pull request by iafisher**

### CODE REVIEW

**Comment by mjwalter** [comment]
Hi Ian - this looks like a good way forward! Feel free to keep developing this locally.

**Comment by mjwalter** [comment]
Because adding a new model means that we would need to perform migrations on the database, it may be wise not to update the repository iteratively, especially because there are presentations coming up. But again, this looks good.

**Comment by iafisher** [comment]
@ajportier @ajportier I've pushed another commit to this PR with the models for the forum feature. As Micah advised I won't be merging this branch anytime soon, but it would be great if you guys could take a look and give me feedback.

File: crim/models/group.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.contrib.auth.models import Group
+from django.db import models
+
+
+class CRIMGroup(Group):
```
Comment (apjanco) [inline crim/models/group.py]: Do we not need a group name?  Users in the group?

**Comment by iafisher** [comment]
@apjanco Yes that's exactly right. The group name is part of the [base Django group model](https://docs.djangoproject.com/en/2.2/ref/contrib/auth/#group-model), while the user-group relationship is part of the [base Django user model](https://docs.djangoproject.com/en/2.2/ref/contrib/auth/#user-model).

**Comment by mjwalter** [comment]
Hi Ian,

This looks good! Two requests that are basically logistical:

* Because all our other CRIM classes begin with `CRIM`, maybe you could rename the classes to `CRIMForumPost` and `CRIMForumComment` for consistency.

* Could you close this pull request and merge it with the `dev` branch instead? That is what the dev site uses.

File: crim/admin.py [hunk 2]
```diff
@@ -682,6 +682,7 @@ class UserAdmin(UserAdmin):
 
 admin.site.unregister(User)
 admin.site.register(User, UserAdmin)
+admin.site.register(CRIMUserProfile)
```
Comment (mjwalter) [inline crim/admin.py]: We don't want to add `admin.site.register(CRIMUserProfile)` -- our custom user profile is automatically linked to the main user account, and the additional details can be updated there. Perhaps we can add a comment explaining this.

File: crim/models/user.py [hunk 3]
```diff
@@ -22,6 +22,7 @@ class Meta:
         'CRIMPerson',
         on_delete=models.SET_NULL,
         null=True,
+        blank=True,
```
Comment (mjwalter) [inline crim/models/user.py]: We don't want both `null=True` and `blank=True`. The first should be sufficient here.

**Pull request closed without merging**